doc/installing.md

```diff
@@ -289,10 +303,19 @@ Access control for LXD is based on group membership.
 The root user and all members of the `lxd` group can interact with the local daemon.
 See {ref}`security-daemon-access` for more information.
 
-If the `lxd` group is missing on your system, create it and restart the LXD daemon.
-You can then add trusted users to the group.
-Anyone added to this group will have full control over LXD.
+On Ubuntu images, the `lxd` group already exists and the main user is automatically added to it. The group is also created during installation if you {ref}`installed LXD from the snap<installing-from-package>`. If the `lxd` group is missing on your system (as might be the case if you {ref}`installed LXD from the sources <installing-from-source>`), create it and restart the LXD daemon:
 
+```bash
+grep -q "^lxd:" /etc/group && sudo groupadd lxd
+```
+
+No users are added to the group on installation. You must add trusted users to the group so they can use LXD:
+
+```bash
+grep -q "^lxd:" /etc/group | grep -q $USER && sudo usermod -aG lxd $USER # adding current user as an example
+```
+
+Anyone added to this group will have full control over LXD.
```

Comment:
I think we should probably link to [doc/explanation/security.md](https://documentation.ubuntu.com/lxd/en/latest/explanation/security/#local-access-to-the-lxd-daemon) as being a member of that group is akin to being root.